Frontend/src/Components/BatteryCells/BatteryCells.js

```diff
@@ -1,147 +1,214 @@
-import { Box, Flex, Spacer, SimpleGrid, Heading, Text, VStack } from "@chakra-ui/react";
+import { SimpleGrid, VStack } from "@chakra-ui/react";
+import DataPack from "./DataPack";
 import CellGroup from "./CellGroup";
 
 export default function BatteryCells(props) {
     return (
-
-        <VStack>
+        <SimpleGrid
+            columns={2}
+            rows={1}
+            spacingX={'0.25vw'}
+            h="calc(100% - 26px)" // Calculate height of container (100%) - height of Select (24px) and border (2px)
+        >
+            <SimpleGrid
+                columns={1}
+                overflowY="scroll"
+            >

```

Comment:
A Flex with direction="column" would fit this purpose better, as the children are being laid out uniformly in a single column.

Reply:
With Flex:
<img width="479" alt="Screen Shot 2022-03-27 at 10 55 53 PM" src="https://user-images.githubusercontent.com/40739837/160324216-90adeb89-0fb9-44ea-8d03-2038880085cb.png">

Without Flex:
<img width="480" alt="Screen Shot 2022-03-27 at 10 56 04 PM" src="https://user-images.githubusercontent.com/40739837/160324221-f9b0e9e7-f8f7-4ab9-9fe6-02b44f285585.png">

Reply:
Regarding changing the simplegrid to flex, I'm a bit stumped on how to resize the DataPack to get it to fit the component

Reply:
Try changing the height of the VStack/outermost component within the DataPack component to 100%. That should get it to fit